controllers/users.js

```diff
@@ -65,7 +67,7 @@ const login = (req, res, next) => {
 
   return User.findUserByCredentials(email, password)
     .then(user => {
-      const token = jwt.sign({ _id: user._id }, NODE_ENV === 'production' ? JWT_SECRET : 'dev-secret');
+      const token = jwt.sign({ _id: user._id }, NODE_ENV === 'production' ? JWT_SECRET : SECRET);
```

Comment:
Можно лучше: вынести настройки в отдельный конфигурационный файл.
Я имел ввиду вынести полностью `NODE_ENV === 'production' ? JWT_SECRET : SECRET`. В управляющем коде нам нужен просто секретный ключ, тут не важно как он был получен. Да и код сократится:
```suggestion
      const token = jwt.sign({ _id: user._id }, JWT_SECRET);
```

Актуально и для:
https://github.com/romcath/praktikum_backend-mesto/blob/15990bb6c12ac89f0efba7e3319600f2546f3658/middlewares/auth.js#L15